pytorch_lightning/plugins/precision/mixed.py

```diff
@@ -11,16 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import TYPE_CHECKING, Union
+from abc import abstractmethod
+from typing import Union
 
 from pytorch_lightning.plugins.precision.precision_plugin import PrecisionPlugin
-
-if TYPE_CHECKING:
-    from pytorch_lightning.utilities import AMPType
+from pytorch_lightning.utilities import AMPType
 
 
 class MixedPrecisionPlugin(PrecisionPlugin):
     """Base Class for mixed precision."""
 
-    backend: "AMPType"
+    @property
+    @abstractmethod
```

Comment:
Why abstractmethod?

```suggestion
```

Reply:
because this should be implemented by all the MixedPrecisionPlugins. But you're right. Without ABC as base, this doesn't work :)